R/modules.R

```diff
@@ -197,7 +197,7 @@ module <- function(label = "module",
     )
   }
   if ("datasets" %in% server_formals) {
-    warning(
+    stop(
       sprintf("Called from module(label = \"%s\", ...)\n  ", label),
```

Comment:
I think we should just remove.
It's been a while. And remove above in the `stop` condition

Reply:
If we remove it we need to provide a support to people using this feature. I know at least one team which uses `datasets` in their modules.

Reply:
I suggest to make an issue to provide a support (when removed)

Reply:
Let's maybe keep it as `stop` and point to an issue that provides the support?
And let's create the instructions on this issue?

Reply:
I think raising the bar from `warning` to `stop` will be enough. The instructions on the warning were long enough to be helpful. I don't think it needs a new issue to explain this.

Reply:
@llrs-roche can you bring back the error-message and use stop, instead of warning?